tests/test_export_onnx.py

```diff
@@ -0,0 +1,149 @@
+from detectron2.structures import Boxes, Instances
+from detectron2.utils.testing import get_sample_coco_image, random_boxes, skip_on_cpu_ci
+
+from ._pytorch_monkey_patches import (
+    _pytorch1111_symbolic_opset9_repeat_interleave,
+    _pytorch1111_symbolic_opset9_to,
+)
+from .helper import (
+    _register_custom_op_onnx_export,
+    _unregister_custom_op_onnx_export,
+    min_torch_version,
+)
+
+
+@unittest.skipIf(not _check_module_exists("onnx"), "ONNX not installed.")
+@unittest.skipIf(not min_torch_version("1.10"), "PyTorch 1.10 is the minimum version")
+class TestONNXTracingExport(unittest.TestCase):
+    def testMaskRCNNFPN(self):
+        def inference_func(model, images):
+            inputs = [{"image": image} for image in images]
+            inst = model.inference(inputs, do_postprocess=False)[0]
+            return [{"instances": inst}]
+
+        self._test_model_zoo_from_config_path(
+            "COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x.yaml", inference_func
+        )
+
+    @skip_on_cpu_ci
+    def testMaskRCNNC4(self):
+        def inference_func(model, image):
+            inputs = [{"image": image}]
+            return model.inference(inputs, do_postprocess=False)[0]
+
+        self._test_model_zoo_from_config_path(
+            "COCO-InstanceSegmentation/mask_rcnn_R_50_C4_3x.yaml", inference_func
+        )
+
+    @skip_on_cpu_ci
+    @unittest.skipIf(not min_torch_version("1.10"), "PyTorch 1.10 is the minimum version")
+    def testCascadeRCNN(self):
+        def inference_func(model, image):
+            inputs = [{"image": image}]
+            return model.inference(inputs, do_postprocess=False)[0]
+
+        self._test_model_zoo_from_config_path(
+            "Misc/cascade_mask_rcnn_R_50_FPN_3x.yaml", inference_func
+        )
+
+    def testRetinaNet(self):
+        def inference_func(model, image):
+            return model.forward([{"image": image}])[0]["instances"]
+
+        self._test_model_zoo_from_config_path(
+            "COCO-Detection/retinanet_R_50_FPN_3x.yaml", inference_func
+        )
+
+    @skip_on_cpu_ci
+    def testMaskRCNNFPN_batched(self):
+        def inference_func(model, image1, image2):
+            inputs = [{"image": image1}, {"image": image2}]
+            return model.inference(inputs, do_postprocess=False)
+
+        self._test_model_zoo_from_config_path(
+            "COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x.yaml", inference_func, batch=2
+        )
+
+    @unittest.skipIf(not min_torch_version("1.11.1"), "PyTorch 1.11.1+ is the minimum version")
+    def testMaskRCNNFPN_with_postproc(self):
+        def inference_func(model, image):
+            inputs = [{"image": image, "height": image.shape[1], "width": image.shape[2]}]
+            return model.inference(inputs, do_postprocess=True)[0]["instances"]
+
+        self._test_model_zoo_from_config_path(
+            "COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x.yaml", inference_func, opset_version=16
+        )
+
```

Comment:
@FrancescoMandru does that answer the question about the post processing?

Reply:
Yes @thiagocrepaldi, thank you!